cpp/include/algorithms.h

```diff
@@ -20,27 +20,31 @@
+ * @Param[in] graph                  cuGRAPH graph descriptor, should contain the connectivity information as an edge list (edge weights are not used for this algorithm).
+ *                                   The transposed adjacency list will be computed if not already present.
+ * @Param[in] alpha                  The damping factor alpha represents the probability to follow an outgoing edge, standard value is 0.85.
  Thus, 1.0-alpha is the probability to “teleport” to a random vertex. Alpha should be greater than 0.0 and strictly lower than 1.0.
- * @Param[in] has_guess           This parameter is used to notify cuGRAPH if it should use a user-provided initial guess. False means the user doesn't have a guess, in this case cuGRAPH will use a uniform vector set to 1/V.
- *                                If the value is True, cuGRAPH will read the pagerank parameter and use this as an initial guess.
- *                                The initial guess must not be the vector of 0s. Any value other than 1 or 0 is treated as an invalid value.
- * @Param[in] pagerank (optional) Initial guess if has_guess=true
- * @Param[in] tolerance           Set the tolerance the approximation, this parameter should be a small magnitude value.
- *                                The lower the tolerance the better the approximation. If this value is 0.0f, cuGRAPH will use the default value which is 1.0E-6.
- *                                Setting too small a tolerance can lead to non-convergence due to numerical roundoff. Usually values between 0.01 and 0.00001 are acceptable.
- * @Param[in] max_iter            The maximum number of iterations before an answer is returned. This can be used to limit the execution time and do an early exit before the solver reaches the convergence tolerance.
- *                                If this value is lower or equal to 0 cuGRAPH will use the default value, which is 500.
- *
- * @Param[out] *pagerank          The PageRank : pagerank[i] is the PageRank of vertex i.
- *
- * @Returns                       GDF_SUCCESS upon successful completion.
+ * @Param[in] has_guess              This parameter is used to notify cuGRAPH if it should use a user-provided initial guess. False means the user doesn't have a guess, in this case cuGRAPH will use a uniform vector set to 1/V.
+ *                                   If the value is True, cuGRAPH will read the pagerank parameter and use this as an initial guess.
+ *                                   The initial guess must not be the vector of 0s. Any value other than 1 or 0 is treated as an invalid value.
+ * @Param[in] pagerank (optional)    Initial guess if has_guess=true
+ * @Param[in] personalization_subset (optional) Vertices for running personalized page rank
+ * @Param[in] personalization_values (optional) Values associated with personalization_subset vertices
```

Comment:
The size is arbitrary correct?

Reply:
I mean a user could potentially personalize everything (size=V) or a single vertex (size=1)?

Reply:
Yes, that's right